tails_files/install.sh

```diff
@@ -209,23 +209,20 @@ function lookup_document_aths_url()
   # The Admin Workstation will certainly have this file, and the Journalist
   # Workstation probably, but not definitely.
   if [ -f "${securedrop_ansible_base}/app-document-aths" ] ; then
-    echo "Found Document Interface ATHS info in '${securedrop_ansible_base}/app-document-aths'..."
     app_document_aths="$(awk '{ print $2 }' ${securedrop_ansible_base}/app-document-aths)"
   # Failing that, check for ATHS info in the existing Tor config. A previously
   # configured Journalist Workstation will already have the value in torrc.
   # Since we know we're in a Journalist Workstation, assume there's only one
   # ATHS value in the torrc, and use the first.
   elif grep -q -P '^HidServAuth [a-z2-7]{16}\.onion [A-Za-z0-9+/.]{22}' /etc/tor/torrc ; then
-    echo "Found Document Interface ATHS info in '/etc/tor/torrc'..."
     app_document_aths="$(grep ^HidServAuth /etc/tor/torrc | head -n 1 | awk '{ print $2 }')"
   # Last shot before prompting: check for an existing desktop icon specifically
   # for the Document Interface. If found, we can extract the URL from there.
   elif [ -e "${amnesia_desktop}/document.desktop" ] ; then
-    echo "Found Document Interface ATHS info in '${amnesia_desktop}/document.desktop'..."
     app_document_aths="$(grep ^Exec=/usr/local/bin/tor-browser | awk '{ print $2 }')"
   # Couldn't find it anywhere. We'll have to prompt!
   else
-    echo "Could not find Document Interface ATHS info, prompting interactively..."
+    echo "Could not find Document Interface ATHS info, prompting interactively..." 1>&2
```

Comment:
By redirecting the verbose logging message to STDERR, we ensure that the echoed text does not interfere with any of the variable lookups. All of the lookup functions use STDOUT capture (via `$()`) to return config information. This is mostly a defensive measure to ensure that outputs are not mixed.